doc/releases/changelog-0.37.0.md

```diff
@@ -20,43 +20,88 @@
+  noise is inserted with the following:
+
+  * `qml.noise.partial_wires(op)`: insert `op` on the wires that are specified by the condition that triggers adding this noise
+  * custom noise operations: custom noise can be specified by defining a standard quantum function like below.
+  
+    ```python
+    def n0(op, **kwargs):
+        qml.RY(op.parameters[0] * 0.05, wires=op.wires)
+    ```
+
+  With that, we can create a `qml.NoiseModel` object whose argument must be a dictionary mapping conditions
+  to noise:
+
+  ```python
+  c1 = qml.noise.op_eq(qml.X) & qml.noise.wires_in([0, 1])
+  n1 = qml.noise.partial_wires(qml.AmplitudeDamping, 0.4)
+
+  noise_model = qml.NoiseModel({c0: n0, c1: n1})
+  ```
+
   ```pycon
-  >>> fcond1 = qml.noise.op_eq(qml.RX) & qml.noise.wires_in([0, 1])
-  >>> noise1 = qml.noise.partial_wires(qml.PhaseDamping, 0.4)
-  >>> fcond2 = qml.noise.op_in([qml.RY, qml.RZ])
-  >>> def noise2(op, **kwargs):
-  ...     qml.ThermalRelaxationError(op.parameters[0] * 0.05, kwargs["t1"], 0.2, 0.6, op.wires)
-  >>> noise_model = qml.NoiseModel({fcond1: noise1, fcond2: noise2}, t1=2.0)
   >>> noise_model
   NoiseModel({
-      OpEq(RX) & WiresIn([0, 1]) = PhaseDamping(gamma=0.4)
-      OpIn(['RY', 'RZ']) = noise2
-  }, t1 = 2.0)
+      BooleanFn(c0): n0
+      OpEq(PauliX) | WiresIn([0, 1]): AmplitudeDamping(gamma=0.4)
+  })
+  ```
+
+  The noise model created can then be added to a QNode with `qml.add_noise`:
+
+  ```python
+  dev = qml.device("lightning.qubit", wires=3)
+
+  @qml.qnode(dev)
+  def circuit():
+      qml.Y(0)
+      qml.CNOT([0, 1])
+      qml.RY(0.3, wires=2) # triggers c0
+      qml.X(1) # triggers c1
+      return qml.state()
   ```
 
   ```pycon
-  >>> @partial(qml.transforms.add_noise, noise_model=noise_model)
-  ... @qml.qnode(dev)
-  ... def circuit(w, x, y, z):
-  ...    qml.RX(w, wires=0)
-  ...    qml.RY(x, wires=1)
-  ...    qml.CNOT(wires=[0, 1])
-  ...    qml.RY(y, wires=0)
-  ...    qml.RX(z, wires=1)
-  ...    return qml.expval(qml.Z(0) @ qml.Z(1))
-  >>> print(qml.draw(circuit)(0.9, 0.4, 0.5, 0.6))
-  0: ──RX(0.90)──PhaseDamping(0.40)──────────────────────────╭●──RY(0.50)
-  1: ──RY(0.40)──ThermalRelaxationError(0.02,2.00,0.20,0.60)─╰X──RX(0.60)
-  ───ThermalRelaxationError(0.03,2.00,0.20,0.60)─┤ ╭<Z@Z>
-  ───PhaseDamping(0.40)──────────────────────────┤ ╰<Z@Z>
+  >>> print(qml.draw(circuit)())
+  0: ──Y────────╭●────┤  State
+  1: ───────────╰X──X─┤  State
+  2: ──RY(0.30)───────┤  State
+  >>> circuit = qml.add_noise(circuit, noise_model)
+  >>> print(qml.draw(circuit)())
+  0: ──Y────────╭●───────────────────────────────────┤  State
+  1: ───────────╰X─────────X──AmplitudeDamping(0.40)─┤  State
+  2: ──RY(0.30)──RY(0.01)────────────────────────────┤  State
   ```
 
+  To learn more about this new functionality, check out our [noise module documentation](https://docs.pennylane.ai/en/stable/code/qml_noise.html)
+  and keep your eyes peeled for an in-depth demo!
```

Comment:
I suspect that this might be a little too much detail to get into here, which is why I linked to the docs. What about just a couple sentences:

```suggestion
  If more than one transform is applied to a QNode, control over when/where the `add_noise` transform is applied 
  in relation to the other transforms can be specified with the `level` keyword argument. By default, `add_noise` is applied
  after all the transforms that have been manually applied to the QNode until that point.
  To learn more about this new functionality, check out our [noise module documentation](https://docs.pennylane.ai/en/stable/code/qml_noise.html)
  and keep your eyes peeled for an in-depth demo!
```

Reply:
@trbromley

Reply:
Thanks, it's great! :+1: